### ISSUE TITLE
Unable to load obs-localvocal.dll

### ISSUE THREAD

**Issue by slhad** [occ-ai/obs-localvocal#142]
Hello,

I am a AMD RX 7900XTX Windows 11 user and installed obs-localvocal with cpu version then blast version, but the dll for Localvocal does not load, do you have any idea for what would prevent it ?

OBS startup logs :
10:38:08.401 timestamp for error msg loading of localvocal

```
10:38:05.604: [Safe Mode] Unclean shutdown detected!
10:38:07.332: [Safe Mode] User elected to launch normally.
10:38:07.332: CPU Name: AMD Ryzen 7 5800X3D 8-Core Processor           
10:38:07.332: CPU Speed: 3400MHz
10:38:07.332: Physical Cores: 8, Logical Cores: 16
10:38:07.332: Physical Memory: 32680MB Total, 13303MB Free
10:38:07.332: Windows Version: 10.0 Build 22631 (release: 23H2; revision: 3880; 64-bit)
10:38:07.332: Running as administrator: true
10:38:07.332: Windows 10/11 Gaming Features:
10:38:07.332: 	Game Bar: Off
10:38:07.332: 	Game DVR: Off
10:38:07.332: 	Game DVR Background Recording: Off
10:38:07.333: 	Game Mode: Probably On (no reg key set)
10:38:07.334: Sec. Software Status:
10:38:07.334: 	Antivirus Microsoft Defender: enabled (AV)
10:38:07.334: 	Pare-feu Windows: enabled (FW)
10:38:07.335: Current Date/Time: 2024-07-23, 10:38:07
10:38:07.335: Browser Hardware Acceleration: true
10:38:07.335: Hide OBS windows from screen capture: false
10:38:07.335: Qt Version: 6.6.3 (runtime), 6.6.3 (compiled)
10:38:07.335: Portable mode: false
10:38:07.376: OBS 30.2.0 (64-bit, windows)
10:38:07.376: ---------------------------------
10:38:07.409: ---------------------------------
10:38:07.409: audio settings reset:
10:38:07.409: 	samples per sec: 48000
10:38:07.409: 	speakers:        2
10:38:07.409: 	max buffering:   960 milliseconds
10:38:07.409: 	buffering type:  dynamically increasing
10:38:07.410: ---------------------------------
10:38:07.410: Initializing D3D11...
10:38:07.410: Available Video Adapters: 
10:38:07.416: 	Adapter 0: AMD Radeon RX 7900 XTX
10:38:07.416: 	  Dedicated VRAM: 25702866944 (23.9 GiB)
10:38:07.416: 	  Shared VRAM:    17134043136 (16.0 GiB)
10:38:07.416: 	  PCI ID:         1002:744c
10:38:07.416: 	  HAGS Status:    Enabled (Default: No, Driver status: Supported)
10:38:07.416: 	  Driver Version: 32.0.11029.1008
10:38:07.417: 	  output 0:
10:38:07.417: 	    name=U28E590
10:38:07.417: 	    pos={0, 0}
10:38:07.417: 	    size={3840, 2160}
10:38:07.417: 	    attached=true
10:38:07.417: 	    refresh=60
10:38:07.417: 	    bits_per_color=10
10:38:07.417: 	    space=RGB_FULL_G22_NONE_P709
10:38:07.417: 	    primaries=[r=(0.633789, 0.340820), g=(0.311523, 0.635742), b=(0.158203, 0.061523), wp=(0.312500, 0.329102)]
10:38:07.417: 	    relative_gamut_area=[709=1.027526, P3=0.757462, 2020=0.543428]
10:38:07.417: 	    sdr_white_nits=80
10:38:07.417: 	    nit_range=[min=0.500000, max=270.000000, max_full_frame=270.000000]
10:38:07.417: 	    dpi=144 (150%)
10:38:07.417: 	    id=\\?\DISPLAY#SAM0C4D#7&37b0fa80&4&UID256#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}
10:38:07.417: 	    alt_id=\\.\DISPLAY17
10:38:07.417: 	  output 1:
10:38:07.417: 	    name=C24F390
10:38:07.417: 	    pos={3840, 654}
10:38:07.417: 	    size={1920, 1080}
10:38:07.417: 	    attached=true
10:38:07.417: 	    refresh=59
10:38:07.417: 	    bits_per_color=8
10:38:07.417: 	    space=RGB_FULL_G22_NONE_P709
10:38:07.417: 	    primaries=[r=(0.645508, 0.336914), g=(0.328125, 0.615234), b=(0.146484, 0.055664), wp=(0.313477, 0.329102)]
10:38:07.417: 	    relative_gamut_area=[709=1.018082, P3=0.750501, 2020=0.538434]
10:38:07.417: 	    sdr_white_nits=80
10:38:07.417: 	    nit_range=[min=0.500000, max=270.000000, max_full_frame=270.000000]
10:38:07.417: 	    dpi=96 (100%)
10:38:07.417: 	    id=\\?\DISPLAY#SAM0D2C#7&37b0fa80&4&UID264#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}
10:38:07.417: 	    alt_id=\\.\DISPLAY18
10:38:07.417: 	  output 2:
10:38:07.417: 	    name=C24F390
10:38:07.417: 	    pos={-1920, 668}
10:38:07.417: 	    size={1920, 1080}
10:38:07.417: 	    attached=true
10:38:07.417: 	    refresh=59
10:38:07.417: 	    bits_per_color=8
10:38:07.417: 	    space=RGB_FULL_G22_NONE_P709
10:38:07.417: 	    primaries=[r=(0.645508, 0.336914), g=(0.328125, 0.615234), b=(0.146484, 0.055664), wp=(0.313477, 0.329102)]
10:38:07.417: 	    relative_gamut_area=[709=1.018082, P3=0.750501, 2020=0.538434]
10:38:07.417: 	    sdr_white_nits=80
10:38:07.417: 	    nit_range=[min=0.500000, max=270.000000, max_full_frame=270.000000]
10:38:07.417: 	    dpi=96 (100%)
10:38:07.417: 	    id=\\?\DISPLAY#SAM0D2C#7&37b0fa80&4&UID268#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}
10:38:07.417: 	    alt_id=\\.\DISPLAY19
10:38:07.417: 	Adapter 1: AMD Radeon RX 7900 XTX
10:38:07.417: 	  Dedicated VRAM: 25702866944 (23.9 GiB)
10:38:07.417: 	  Shared VRAM:    17134043136 (16.0 GiB)
10:38:07.417: 	  PCI ID:         1002:744c
10:38:07.417: 	  HAGS Status:    Enabled (Default: No, Driver status: Supported)
10:38:07.417: 	  Driver Version: 32.0.11029.1008
10:38:07.418: Loading up D3D11 on adapter AMD Radeon RX 7900 XTX (0)
10:38:07.452: D3D11 loaded successfully, feature level used: b000
10:38:07.452: DXGI increase maximum frame latency success
10:38:07.452: Hardware-Accelerated GPU Scheduling enabled on adapter!
10:38:07.452: D3D11 GPU priority setup success
10:38:07.532: ---------------------------------
10:38:07.532: video settings reset:
10:38:07.532: 	base resolution:   2560x1440
10:38:07.532: 	output resolution: 2560x1440
10:38:07.532: 	downscale filter:  Bicubic
10:38:07.532: 	fps:               60/1
10:38:07.532: 	format:            NV12
10:38:07.532: 	YUV mode:          Rec. 709/Partial
10:38:07.532: NV12 texture support enabled
10:38:07.532: P010 texture support not available
10:38:07.533: Audio monitoring device:
10:38:07.533: 	name: Default
10:38:07.533: 	id: default
10:38:07.533: ---------------------------------
10:38:07.535: Skipping module '../../obs-plugins/64bit/advanced-scene-switcher-lib.dll', not an OBS plugin
10:38:07.535: Skipping module '../../obs-plugins/64bit/advanced-scene-switcher-opencv.dll', not an OBS plugin
10:38:07.535: Skipping module '../../obs-plugins/64bit/advanced-scene-switcher-openvr.dll', not an OBS plugin
10:38:07.539: [adv-ss] version: 1.23.1
10:38:07.539: [adv-ss] version: 68c6492c3f4e5507a343bd821f7112610d32e430
10:38:07.540: [adv-ss] curl loaded successfully
10:38:07.540: [adv-ss] found curl library
10:38:07.540: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-midi.dll"
10:38:07.541: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-midi.dll"
10:38:07.541: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-opencv.dll"
10:38:07.545: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-opencv.dll"
10:38:07.545: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-openvr.dll"
10:38:07.546: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-openvr.dll"
10:38:07.546: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_calib3d453.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_calib3d453.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_calib3d460.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_calib3d460.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_core453.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_core453.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_core460.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_core460.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_features2d453.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_features2d453.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_features2d460.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_features2d460.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_flann453.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_flann453.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_flann460.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_flann460.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_imgproc453.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_imgproc453.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_imgproc460.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_imgproc460.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_objdetect453.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_objdetect453.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_objdetect460.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_objdetect460.dll"
10:38:07.549: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/openvr_api.dll"
10:38:07.549: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/openvr_api.dll"
10:38:07.551: QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 0,0,0,0'
10:38:07.551: QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 0,0,0,0'
10:38:07.551: QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 0,0,0,0'
10:38:07.553: QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 0,0,0,0'
10:38:07.567: No AJA devices found, skipping loading AJA UI plugin
10:38:07.567: Failed to initialize module 'aja-output-ui.dll'
10:38:07.568: No AJA devices found, skipping loading AJA plugin
10:38:07.568: Failed to initialize module 'aja.dll'
10:38:07.568: [Audio Monitor] loaded version 0.8.2
10:38:07.580: VolumeMeter(0x1d1ea59f8d0)  does not have a property named  "meterThickness"
10:38:07.581: VolumeMeter(0x1d190b1aa60)  does not have a property named  "meterThickness"
10:38:07.582: VolumeMeter(0x1d1ea59f8d0)  does not have a property named  "meterThickness"
10:38:07.582: VolumeMeter(0x1d190b1aa60)  does not have a property named  "meterThickness"
10:38:07.586: Skipping module '../../obs-plugins/64bit/chrome_elf.dll', not an OBS plugin
10:38:07.722: [CoreAudio encoder]: Adding CoreAudio AAC encoder
10:38:07.723: Skipping module '../../obs-plugins/64bit/ctranslate2.dll', not an OBS plugin
10:38:07.723: Failed to load 'en-US' text for module: 'decklink-captions.dll'
10:38:07.724: Failed to load 'en-US' text for module: 'decklink-output-ui.dll'
10:38:07.726: A DeckLink iterator could not be created.  The DeckLink drivers may not be installed
10:38:07.726: Failed to initialize module 'decklink.dll'
10:38:07.727: [Downstream Keyer] loaded version 0.2.6
10:38:08.055: [AMF] Version 2.8.0 loaded (Compiled: 1.4.14.0, Runtime: 1.4.34.0, Library: 1.4.34.0;24.10.29.01;AMD-Radeon-Driver/drivers:9fd3d48a716c780b224cd1a22044381f4cc194cc).
10:38:08.106: [AMF] <Id: 1> Unable to set converter transfer characteristic, error AMF_NOT_FOUND (code 11)
10:38:08.154: [AMF] <Id: 2> Unable to set converter transfer characteristic, error AMF_NOT_FOUND (code 11)
10:38:08.155: [AMF] [Capability Manager] Testing Direct3D 11 Adapter 'AMD Radeon RX 7900 XTX (VEN_1002/DEV_744c/SUB_e3b1002/REV_00c8)':
10:38:08.155:   H264/AVC: Supported
10:38:08.155:   H265/HEVC: Supported
10:38:08.155: 
10:38:08.207: [AMF] <Id: 3> Unable to set converter transfer characteristic, error AMF_NOT_FOUND (code 11)
10:38:08.258: [AMF] <Id: 4> Unable to set converter transfer characteristic, error AMF_NOT_FOUND (code 11)
10:38:08.259: [AMF] [Capability Manager] Testing Direct3D 11 Adapter 'AMD Radeon RX 7900 XTX (VEN_1002/DEV_744c/SUB_e3b1002/REV_00c8)':
10:38:08.259:   H264/AVC: Supported
10:38:08.259:   H265/HEVC: Supported
10:38:08.259: 
10:38:08.266: [Gradient Source] loaded version 0.3.0
10:38:08.267: Skipping module '../../obs-plugins/64bit/libcef.dll', not an OBS plugin
10:38:08.267: Skipping module '../../obs-plugins/64bit/libEGL.dll', not an OBS plugin
10:38:08.267: Skipping module '../../obs-plugins/64bit/libfftw3-3.dll', not an OBS plugin
10:38:08.267: Skipping module '../../obs-plugins/64bit/libGLESv2.dll', not an OBS plugin
10:38:08.267: Skipping module '../../obs-plugins/64bit/libopenblas.dll', not an OBS plugin
10:38:08.270: [Move Transition] loaded version 2.9.4
10:38:08.270: Failed to get nvARPose.dll version info size
10:38:08.281: [obs-browser]: Version 2.23.5
10:38:08.281: [obs-browser]: CEF Version 103.0.5060.134 (runtime), 103.0.0-5060-shared-textures.2594+gc69ad37+chromium-103.0.5060.134 (compiled)
10:38:08.282: [Composite Blur] loaded version 1.0.2
10:38:08.398: [noise suppress]: NVIDIA denoiser disabled, redistributable not found or could not be loaded.
10:38:08.399: Failed to get NVVideoEffects.dll version info size
10:38:08.400: [NVIDIA VIDEO FX]: FX disabled, redistributable not found or could not be loaded.
10:38:08.401: LoadLibrary failed for '../../obs-plugins/64bit/obs-localvocal.dll': The specified module could not be found.
10:38:08.401:  (126)
10:38:08.401: Module '../../obs-plugins/64bit/obs-localvocal.dll' not loaded
10:38:08.460: [obs-roi-ui] plugin loaded successfully (version 1.1.1)
10:38:08.465: [obs-websocket] [obs_module_load] you can haz websockets (Version: 5.5.1 | RPC Version: 1)
10:38:08.465: [obs-websocket] [obs_module_load] Qt version (compile-time): 6.6.3 | Qt version (run-time): 6.6.3
10:38:08.465: [obs-websocket] [obs_module_load] Linked ASIO Version: 103002
10:38:08.471: [obs-websocket] [obs_module_load] Module loaded.
10:38:08.472: Skipping module '../../obs-plugins/64bit/onnxruntime.dll', not an OBS plugin
10:38:08.472: Skipping module '../../obs-plugins/64bit/onnxruntime_providers_shared.dll', not an OBS plugin
10:38:08.472: Skipping module '../../obs-plugins/64bit/opencv_calib3d453.dll', not an OBS plugin
10:38:08.472: Skipping module '../../obs-plugins/64bit/opencv_core453.dll', not an OBS plugin
10:38:08.472: Skipping module '../../obs-plugins/64bit/opencv_features2d453.dll', not an OBS plugin
10:38:08.472: Skipping module '../../obs-plugins/64bit/opencv_flann453.dll', not an OBS plugin
10:38:08.472: Skipping module '../../obs-plugins/64bit/opencv_imgproc453.dll', not an OBS plugin
10:38:08.473: Skipping module '../../obs-plugins/64bit/opencv_objdetect453.dll', not an OBS plugin
10:38:08.473: Skipping module '../../obs-plugins/64bit/openvr_api.dll', not an OBS plugin
10:38:08.473: [Recursion Effect] loaded version 0.0.4
10:38:08.475: [Scene As Transition] loaded version 1.0.2
10:38:08.476: [Scene Collection Manager] loaded version 0.0.8
10:38:08.477: [Scene Notes Dock] loaded version 0.1.1
10:38:08.486: QLayout: Attempting to add QLayout "" to SceneNotesDock "SceneNotesDock", which already has a layout
10:38:08.489: [Source Dock] loaded version 0.3.4
10:38:08.489: [Source Record] loaded version 0.3.1
10:38:08.490: [Source Switcher] loaded version 0.4.1
10:38:08.491: [spectralizer] Loading v1.3.4 build time 2021.01.28 20:03
10:38:08.492: [StreamUP Record Chapter Manager] loaded version 1.0.2
10:38:08.521: [StreamUP Record Chapter Manager] Settings loaded successfully from C:\Users\slash\AppData\Roaming\obs-studio/plugin_config/streamup-record-chapter-manager/configs.json
10:38:08.523: [Transition Table] loaded version 0.2.6
10:38:08.524: [tuna] Loading v1.9.1 (build time 2022.10.27 17:50). Qt version: compile-time: 6.3.1, run-time: 6.6.3. libobs: compile-time: 28.0.0, run-time: 30.2.0
10:38:08.543: QObject::connect: Parentheses expected, slot tuna_gui::cb_try_download_cover_clicked
10:38:08.543: QObject::connect:  (sender name:   'cb_dl_cover')
10:38:08.543: QObject::connect:  (receiver name: 'tuna_gui')
10:38:08.543: QObject::connect: Parentheses expected, slot tuna_gui::cb_download_missing_covers_clicked
10:38:08.543: QObject::connect:  (sender name:   'cb_download_missing')
10:38:08.543: QObject::connect:  (receiver name: 'tuna_gui')
10:38:08.563: QMetaObject::connectSlotsByName: No matching signal for on_scene_changed(int)
10:38:08.563: QMetaObject::connectSlotsByName: No matching signal for on_add_source()
10:38:08.563: QMetaObject::connectSlotsByName: No matching signal for on_remove_source()
10:38:08.568: [tuna] Logged into Spotify.
10:38:08.569: [tuna] Could not find 'C:\Users\slash\AppData\Roaming\obs-studio/plugin_config/tuna/tuna_vlc_mappings.json' trying old location 'C:\Users\slash\tuna_vlc_mappings.json'
10:38:08.580: [tuna] Loaded 3 outputs
10:38:08.580: [tuna] Logged into Spotify.
10:38:08.580: [tuna] Could not find 'C:\Users\slash\AppData\Roaming\obs-studio/plugin_config/tuna/tuna_vlc_mappings.json' trying old location 'C:\Users\slash\tuna_vlc_mappings.json'
10:38:08.583: [Vertical Canvas] loaded version 1.4.6
10:38:08.584: [vlc-video]: VLC 3.0.20 Vetinari found, VLC video source enabled
10:38:08.585: [waveform]: Registered v1.0.0 64-bit
10:38:08.585: [waveform]: Using CPU capabilities: AVX2 AVX SSE4.1 FMA3 SSE2
10:38:08.586: Failed to load 'en-US' text for module: 'win-capture-audio.dll'
10:38:08.586: [win-capture-audio] Version 2.2.3 (74323e6)
10:38:08.600: QWidget::setLayout: Attempting to set QLayout "" on QDockWidget "", which already has a layout
10:38:08.601: <StreamDeck> Plugin version 5.5.2.16
10:38:08.602: <StreamDeck> [Server] Listening on '127.0.0.1:28186'.
10:38:08.602: ---------------------------------
10:38:08.602:   Loaded Modules:
10:38:08.602:     StreamDeckPlugin.dll
10:38:08.602:     obs-multi-rtmp.dll
10:38:08.602:     win-wasapi.dll
10:38:08.602:     win-dshow.dll
10:38:08.602:     win-capture.dll
10:38:08.602:     win-capture-audio.dll
10:38:08.602:     waveform.dll
10:38:08.602:     vlc-video.dll
10:38:08.602:     vertical-canvas.dll
10:38:08.602:     tuna.dll
10:38:08.602:     transition-table.dll
10:38:08.602:     text-freetype2.dll
10:38:08.602:     streamup-record-chapter-manager.dll
10:38:08.602:     spectralizer.dll
10:38:08.602:     source-switcher.dll
10:38:08.602:     source-record.dll
10:38:08.602:     source-dock.dll
10:38:08.602:     scene-notes-dock.dll
10:38:08.602:     scene-collection-manager.dll
10:38:08.602:     scene-as-transition.dll
10:38:08.602:     rtmp-services.dll
10:38:08.602:     recursion-effect.dll
10:38:08.602:     obs-x264.dll
10:38:08.602:     obs-websocket.dll
10:38:08.602:     obs-webrtc.dll
10:38:08.602:     obs-vst.dll
10:38:08.602:     obs-transitions.dll
10:38:08.602:     obs-text.dll
10:38:08.602:     obs-roi-ui.dll
10:38:08.602:     obs-qsv11.dll
10:38:08.602:     obs-outputs.dll
10:38:08.602:     obs-filters.dll
10:38:08.602:     obs-ffmpeg.dll
10:38:08.602:     obs-composite-blur.dll
10:38:08.602:     obs-browser.dll
10:38:08.602:     move-transition.dll
10:38:08.602:     logi_obs_plugin_x64.dll
10:38:08.602:     image-source.dll
10:38:08.602:     gradient-source.dll
10:38:08.602:     frontend-tools.dll
10:38:08.602:     enc-amf.dll
10:38:08.602:     downstream-keyer.dll
10:38:08.602:     decklink-output-ui.dll
10:38:08.602:     decklink-captions.dll
10:38:08.602:     coreaudio-encoder.dll
10:38:08.602:     audio-monitor.dll
10:38:08.602:     advanced-scene-switcher.dll
10:38:08.602: ---------------------------------
10:38:08.602: [Vertical Canvas] Loaded configuration file
10:38:08.656: QLayout: Attempting to add QLayout "" to CanvasDock "contextContainer", which already has a layout
10:38:08.870: [obs-websocket] [obs_module_post_load] WebSocket server is enabled, starting...
10:38:08.870: [obs-websocket] [WebSocketServer::Start] Not locked to IPv4 bindings
10:38:08.871: [obs-websocket] [WebSocketServer::ServerRunner] IO thread started.
10:38:08.874: [obs-websocket] [WebSocketServer::Start] Server started successfully on port 4455. Possible connect address: 192.168.0.118
10:38:08.877: ---------------------------------
10:38:08.877: Available Encoders:
10:38:08.877:   Video Encoders:
10:38:08.877: 	- ffmpeg_svt_av1 (SVT-AV1)
10:38:08.877: 	- ffmpeg_aom_av1 (AOM AV1)
10:38:08.877: 	- h264_texture_amf (AMD HW H.264 (AVC))
10:38:08.877: 	- h265_texture_amf (AMD HW H.265 (HEVC))
10:38:08.877: 	- av1_texture_amf (AMD HW AV1)
10:38:08.877: 	- obs_x264 (x264)
10:38:08.877:   Audio Encoders:
10:38:08.877: 	- CoreAudio_AAC (CoreAudio AAC)
10:38:08.877: 	- ffmpeg_aac (FFmpeg AAC)
10:38:08.877: 	- ffmpeg_opus (FFmpeg Opus)
10:38:08.877: 	- ffmpeg_pcm_s16le (FFmpeg PCM (16-bit))
10:38:08.877: 	- ffmpeg_pcm_s24le (FFmpeg PCM (24-bit))
10:38:08.877: 	- ffmpeg_pcm_f32le (FFmpeg PCM (32-bit float))
10:38:08.877: 	- ffmpeg_alac (FFmpeg ALAC (24-bit))
10:38:08.877: 	- ffmpeg_flac (FFmpeg FLAC (16-bit))
10:38:08.877: ==== Startup complete ===============================================
```

**Comment by royshil** [comment]
```
10:38:08.401: LoadLibrary failed for '../../obs-plugins/64bit/obs-localvocal.dll': The specified module could not be found.
10:38:08.401:  (126)
10:38:08.401: Module '../../obs-plugins/64bit/obs-localvocal.dll' not loaded
```
must have to do with missing DLLs
are you using the installer?

**Comment by slhad** [comment]
Yes, I have used the installer in both case and the file is in directory

```
C:\Program Files\obs-studio>ls -hl obs-plugins\64bit\obs-localvocal.dll
-rwxr-xr-x 1 slash slash 1.2M Jul 19 18:18 'obs-plugins\64bit\obs-localvocal.dll'
```

**Comment by slhad** [comment]
I redownloaded the archive for obs-localvocal-0.3.3-windows-x64-cudaclblast.zip and tried it in a portable OBS folder.
It was working .... so I compared log files from my installed OBS and portable one, only difference was no more error.

Re extracted zip to installed OBS and ... it worked ...

I had still my old archive from 5 days before, same name, different size ...
Difference is OpenCL.dll and whisper.dll are present in the most recent one archive with same name.

Thanks you for fixing the zip file @royshil , maybe a new version or a warning in release notes would have been a good thing.
